### ISSUE TITLE
TypeError: arg.value is not a function

### ISSUE THREAD

**Issue by lijy91** [typicode/lowdb#89]
The error occurred 0.12.1 

https://github.com/lijy91/alfred-coding-workflow/blob/dev/main.js

```
/Users/Lijy91/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/user.workflow.060322F2-E9AE-4758-BCDE-39FFC36C4D10/node_modules/lowdb/index.js:23
        v = arg.value ? arg.value() : arg;
                            ^

TypeError: arg.value is not a function
    at LodashWrapper.<anonymous> (/Users/Lijy91/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/user.workflow.060322F2-E9AE-4758-BCDE-39FFC36C4D10/node_modules/lowdb/index.js:23:29)
    at LodashWrapper.find (/Users/Lijy91/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/user.workflow.060322F2-E9AE-4758-BCDE-39FFC36C4D10/node_modules/lowdb/node_modules/lodash/lodash.js:4104:35)
    at Object.<anonymous> (/Users/Lijy91/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/user.workflow.060322F2-E9AE-4758-BCDE-39FFC36C4D10/main.js:129:46)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:429:10)
    at startup (node.js:139:18)
    at node.js:999:3

```

**Comment by lijy91** [comment]
Field name can not be `key`?

**Comment by typicode** [comment]
Hi,

Thanks for the issue, it seems to be a bug.
If a property is named `value` then it will fail:

```js
const low = require('lowdb')
const db = low()

db('foo').push({a: 1, value: 2})
db('foo').find({a: 1}).value // error!
```

**Comment by typicode** [comment]
I think it should be good now, fixed in `v0.12.2`.

BTW in your code with lowdb `v0.12`, you need to explicitly set a storage:

```js
const low = require('lowdb')
const db = low('db.json') // won't work

const storage = require('lowdb/file-sync')
const db = low('db.json', { storage })

**Comment by lijy91** [comment]
@typicode Thank you, has been working
https://github.com/lijy91/alfred-coding-workflow/blob/master/main.js
